src/utils.ts

```diff
@@ -0,0 +1,16 @@
+import * as core from '@actions/core';
+import { Options } from './validate';
+
+export function isNullOrEmpty(string: string | undefined | null): boolean {
+    return !string || string.length === 0;
```

Comment:
`!string` already is true for an empty string so we don't need to check for its length

Reply:
Correct! Good catch 👏

Interestingly this was a GitHub Copilot assisted function, lesson learned, don't trust the robots 😄🤖

Reply:
Trust but verify 😄